src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_3_horizontal_layout.html.twig

```diff
@@ -27,7 +27,7 @@ col-sm-2
     {%- if help is not empty -%}
         {%- set widget_attr = {attr: {'aria-describedby': id ~"_help"}} -%}
     {%- endif -%}
-    <div class="form-group{% if (not compound or force_error|default(false)) and not valid %} has-error{% endif %}">
+    <div {% with {attr: row_attr.class is defined ? row_attr : row_attr|default({})|merge({class: 'form-group' ~ ((not compound or force_error|default(false)) and not valid ? ' has-error')})} %}{{ block('attributes') }}{% endwith %}>
```

Comment:
so this means that if I want to add a row_attr class (to apply a JS plugin for instance), I have to provide the classes normally part of the theme explicitly ? This makes the theme much harder to use. I suggest merging classes instead.

Reply:
Yes. I thought about it but if we always merge defaults, then you cannot get rid of them if you want to. So it's "redefine everything if you wanna add one but you are able to remove the defaults" vs "merge but the defaults will always be here".

Reply:
@stof Just looked at this again and your suggestion is better for DX. Let's do the same thing that for the label attr -> merge classes but put the default classes in a new `form_row_class` block so it can be extended easily (to remove them for example). Does that work for you?